modules/customize-certificates-api-add-named.adoc

```diff
@@ -113,3 +113,8 @@ kube-apiserver   {product-version}.0     True        False         False      14
 ----
 +
 If `PROGRESSING` is showing `True`, wait a few minutes and try again.
+
+[NOTE]
+====
+The new revision of the Kubernetes API server will only roll out if the API Server named certificate is added for the first time. At the time of renewing the API Server named certificate, the new revision of the Kubernetes API server will not roll out, as the `kube-apiserver` pods dynamically reload the updated certificate.
```

Comment:
A few additional suggestions, if you can please update:

```suggestion
A new revision of the Kubernetes API server only rolls out if the API server named certificate is added for the first time. When the API server named certificate is renewed, a new revision of the Kubernetes API server does not roll out, because the `kube-apiserver` pods dynamically reload the updated certificate.
```